### DISCUSSION TITLE
Drawings on top of a code block

### DISCUSSION THREAD

**Discussion by pgiannikos** [zsviczian/obsidian-excalidraw-plugin#2114] [Q&A]
I'm trying to apply annotations (arrows, shapes, etc) and marker drawings on the top of a code block and it does not seem to work. Any ideas how to eventually do it?

Thanks 

![image](https://github.com/user-attachments/assets/2628d94d-504d-4696-bf9f-61e8c7e15108)

**Comment by LeonelRFF** [comment]
You can embed it as an image and that way yes, although if you want to have it as it is there, a trick you can do is increase the transparency and you can see the drawings and elements.

![Screenshot_2024-12-11-23-34-38-83_51606159b24eff83e24a54116878fe3e](https://github.com/user-attachments/assets/837c4f88-5ead-4cda-ba9b-9cb6a636645d)

**Comment by TheFlowCH** [comment]
I'm running in the same issue. I use Chord Sheets plugin for my guitar tabs, which use code blocks to display tabs based on markdown file. 
I would love to place my tab files in an excalidraw drawing as an Interactive-Frame, in order to annotate them. But I can't find a way to write _over_ the codeblock, any type of writing, and even drawings (squares etc.) go under. I tried playing with frames order to no avail. 
Changing opacity can be a temporary workaround but is not very satisfying as colors get dimmed out. 
Is this a bug or is there a workaround I didn't find?
Thanks in advance
